src/App.js

```diff
@@ -26,7 +31,38 @@ class App extends React.Component {
 	 
 	changedHandler(e) {
         this.setState({username: e.target.value});
-    }
+	}
+	
+	searchForUser() {
+		if (this.state.username === "") {
```

Comment:
You already created a Search Input component, why would you write it's functions and state in the app module?!
Move all logic related to that component INSIDE that component

Reply:
because i want to use the search results outside of the search component and i dont know how to deliver data outside a component to the parent.